tests/unit/s2n_async_pkey_test.c

```diff
@@ -41,6 +41,10 @@ uint8_t offload_callback_count = 0;
 
 typedef int (async_handler)(struct s2n_connection *conn);
 
+/* Declaring a flag to check if sign operation is called at least once for all cipher_suites
+   while performing handshake through handler (async_handler_sign_with_different_pkey_and_apply) */
```

Comment:
Nit: In S2N, we use a comment format that makes it clearer every line is a comment. Please match the style of other S2N block comments.
```suggestion
/* Declaring a flag to check if sign operation is called at least once for all cipher_suites
 * while performing handshake through handler (async_handler_sign_with_different_pkey_and_apply) */
```